gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +0,0 @@
-#Tue Nov 07 13:26:56 CET 2017
-distributionBase=GRADLE_USER_HOME
```

Comment:
I believe this file is also autogenerated as well; we might need to add this file to gitignore otherwise it will be accidentally checked in by a dev working on the SDK. Same for file below.

Reply:
See line 181 in .gitignore in this review (top file).

Reply:
👍